In case it is still useful, here's my `nix-review` output on 20.03:

```
$ nix-review pr --checkout commit 83564
...
error: build of '/nix/store/zkagvwpddw362z15yi7fak541qyxa4j2-python3.8-pywinrm-0.4.1.drv' failed: builder for '/nix/store/zkagvwpddw362z15yi7fak541qyxa4j2-python3.8-pywinrm-0.4.1.drv' failed with exit code 1
builder for '/nix/store/zkagvwpddw362z15yi7fak541qyxa4j2-python3.8-pywinrm-0.4.1.drv' failed with exit code 1; last 10 log lines:
  ModuleNotFoundError: No module named 'winkerberos'
  
  
  ----------------------------------------------------------------------
  Ran 25 tests in 0.014s
  
  FAILED (errors=2)
  Test failed: <unittest.runner.TextTestResult run=25 errors=2 failures=0>
  error: Test failed: <unittest.runner.TextTestResult run=25 errors=2 failures=0>
  builder for '/nix/store/zkagvwpddw362z15yi7fak541qyxa4j2-python3.8-pywinrm-0.4.1.drv' failed with exit code 1
error: build of '/nix/store/ygm1hpsd8yhkdbrdbckva9hgpx9lmd5n-python3.7-pywinrm-0.4.1.drv' failed: builder for '/nix/store/ygm1hpsd8yhkdbrdbckva9hgpx9lmd5n-python3.7-pywinrm-0.4.1.drv' failed with exit code 1
builder for '/nix/store/ygm1hpsd8yhkdbrdbckva9hgpx9lmd5n-python3.7-pywinrm-0.4.1.drv' failed with exit code 1; last 10 log lines:
  ModuleNotFoundError: No module named 'winkerberos'
  
  
  ----------------------------------------------------------------------
  Ran 25 tests in 0.029s
  
  FAILED (errors=2)
  Test failed: <unittest.runner.TextTestResult run=25 errors=2 failures=0>
  error: Test failed: <unittest.runner.TextTestResult run=25 errors=2 failures=0>
  builder for '/nix/store/ygm1hpsd8yhkdbrdbckva9hgpx9lmd5n-python3.7-pywinrm-0.4.1.drv' failed with exit code 1
cannot build derivation '/nix/store/afsihvvkwx4j7h6wlx9b4qfv0095azvy-env.drv': 2 dependencies couldn't be built
[11 built (2 failed), 202 copied (450.4 MiB), 28.5 MiB DL]
error: build of '/nix/store/afsihvvkwx4j7h6wlx9b4qfv0095azvy-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/83564
7 package marked as broken and skipped:
python27Packages.exchangelib python37Packages.exchangelib python37Packages.qasm2image python37Packages.qiskit python38Packages.exchangelib python38Packages.qasm2image python38Packages.qiskit

2 package failed to build:
python37Packages.pywinrm python38Packages.pywinrm

10 package built:
python27Packages.IBMQuantumExperience python27Packages.ntlm-auth python27Packages.pywinrm python27Packages.requests_ntlm python37Packages.IBMQuantumExperience python37Packages.ntlm-auth python37Packages.requests_ntlm python38Packages.IBMQuantumExperience python38Packages.ntlm-auth python38Packages.requests_ntlm
```

Detailed logs for `python3.8-pywinrm-0.4.1` say `No module named 'kerberos'`, so probably unrelated:

<details>

```
winrm.vendor.requests_kerberos (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: requests_kerberos (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: requests_kerberos
Traceback (most recent call last):
  File "/build/pywinrm-0.4.1/winrm/vendor/requests_kerberos/kerberos_.py", line 18, in <module>
    import kerberos
ModuleNotFoundError: No module named 'kerberos'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nix/store/11w4z4w7lszgflgb8j62bwqr15j5zm5a-python3-3.8.1/lib/python3.8/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/build/pywinrm-0.4.1/winrm/vendor/requests_kerberos/__init__.py", line 33, in <module>
    from .kerberos_ import HTTPKerberosAuth, REQUIRED, OPTIONAL, DISABLED
  File "/build/pywinrm-0.4.1/winrm/vendor/requests_kerberos/kerberos_.py", line 20, in <module>
    import winkerberos as kerberos
ModuleNotFoundError: No module named 'winkerberos'


======================================================================
ERROR: winrm.vendor.requests_kerberos (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: winrm.vendor.requests_kerberos
Traceback (most recent call last):
  File "/build/pywinrm-0.4.1/winrm/vendor/requests_kerberos/kerberos_.py", line 18, in <module>
    import kerberos
ModuleNotFoundError: No module named 'kerberos'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nix/store/11w4z4w7lszgflgb8j62bwqr15j5zm5a-python3-3.8.1/lib/python3.8/unittest/loader.py", line 470, in _find_test_path
    package = self._get_module_from_name(name)
  File "/nix/store/11w4z4w7lszgflgb8j62bwqr15j5zm5a-python3-3.8.1/lib/python3.8/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/build/pywinrm-0.4.1/winrm/vendor/requests_kerberos/__init__.py", line 33, in <module>
    from .kerberos_ import HTTPKerberosAuth, REQUIRED, OPTIONAL, DISABLED
  File "/build/pywinrm-0.4.1/winrm/vendor/requests_kerberos/kerberos_.py", line 20, in <module>
    import winkerberos as kerberos
ModuleNotFoundError: No module named 'winkerberos'


----------------------------------------------------------------------
Ran 25 tests in 0.014s

FAILED (errors=2)
Test failed: <unittest.runner.TextTestResult run=25 errors=2 failures=0>
error: Test failed: <unittest.runner.TextTestResult run=25 errors=2 failures=0>
builder for '/nix/store/zkagvwpddw362z15yi7fak541qyxa4j2-python3.8-pywinrm-0.4.1.drv' failed with exit code 1
```

</details>

Same for the 3.7 version.

`pywinrm` did build though on https://github.com/NixOS/nixpkgs/pull/83560#issuecomment-605378644.

--- a/pkgs/development/python-modules/ntlm-auth/default.nix
+++ b/pkgs/development/python-modules/ntlm-auth/default.nix
@@ -3,22 +3,23 @@
 , fetchFromGitHub
 , mock
 , pytest
+, requests
 , unittest2
 , six
 }:
 
 buildPythonPackage rec {
   pname = "ntlm-auth";
-  version = "1.0.3";
+  version = "1.4.0";
 
   src = fetchFromGitHub {
     owner = "jborean93";
     repo = "ntlm-auth";
     rev = "v${version}";
-    sha256 = "09f2g4ivfi9lh1kr30hlg0q4n2imnvmd79w83gza11q9nmhhiwpz";
+    sha256 = "168k3ygwbvnfcwn7q1nv3vvy6b9jc4cnpix0xgg5j8av7v1x0grn";
   };
 
-  checkInputs = [ mock pytest unittest2 ];
+  checkInputs = [ mock pytest requests unittest2 ];
   propagatedBuildInputs = [ six ];
 
   # Functional tests require networking
@@ -28,8 +29,8 @@ buildPythonPackage rec {
 
   meta = with lib; {
     description = "Calculates NTLM Authentication codes";
-    homepage = https://github.com/jborean93/ntlm-auth;
-    license = licenses.lgpl3;
+    homepage = "https://github.com/jborean93/ntlm-auth";
+    license = licenses.mit;
     maintainers = with maintainers; [ elasticdog ];
     platforms = platforms.all;
   };

